src/Compilers/Core/Portable/InternalUtilities/Debug.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Roslyn.Utilities
+{
+    internal static class RoslynDebug
```

Comment:
>RoslynDebug [](start = 26, length = 11)

What if we named this type `Debug`? Are there other members we'd have to implement?

Reply:
I'm a bit loath to call it `Debug` and keep it in the namespace `Roslyn.Utilities`. Makes it really hard to understand which one we're using at a casual glance of `Debug.Assert`. That will make tracking down nullable warnings even harder.

Reply:
Then you'd have conflicts in files that have `using System.Diagnostics;`

Reply:
We can't call it `Debug` because type references to unimported namespaces is a thing.

Reply:
@jcouv Can't do it easily because we use ConditionalAttribute everywhere and it's in the same namespace.